media/slides/index.html

```diff
@@ -28,6 +28,14 @@ <h2>Slides</h2>
                 </span>
                 </a>
                 <p class="text-center media-block-title">OMERO Users Training Day, December 2017 (PDF)</p>
+            </div>
+            <div class="media-block column image-hover-wrapper">
+                <a href="http://downloads.openmicroscopy.org/presentations/2017/Cambridge-Training/Image-Processing-OMERO/#/"><img src="{{ site.baseurl }}/media/slides/thumbnails/2017-12-image-processing-with-omero.png" alt="Image Processing with OMERO">
+                <span class="image-hover-wrapper-reveal">
+                    <p>Download slides<br><i class="fa fa-download" aria-hidden="true"></i></p>
```

Comment:
Can you change this to 'View slides' with a different icon? Download looks odd when it's actually a hyperlink